src/Wrapper.js

```diff
@@ -10,7 +13,7 @@ import MenuPanel from './components/MenuPanel/MenuPanel'
 import SignerPanelContent from './components/SignerPanel/SignerPanelContent'
 import { getAppPath, staticApps } from './routing'
 
-import { groups } from './demo-state'
+// import { groups } from './demo-state'
```

Comment:
Should it be commented out or removed?

Reply:
I guess we could remove this entirely. When permissions will be activated, a demo state won’t be needed anymore 🙌